src/locales/en.json

```diff
@@ -87,6 +87,8 @@
     "cancel": "Cancel",
     "close": "Close",
     "create": "Create",
+    // This is the text of a button that is used to fill out a new Submission.
```

Comment:
Might be helpful to say that it appears next to a headline that says "Submissions". Otherwise folks might be tempted to be helpful and add more text.